### DISCUSSION TITLE
Can you connect a cable/circuit between multiple tenants?

### DISCUSSION THREAD

**Discussion by TheDirkMelvin** [netbox-community/netbox#7903] [Q&A]
We have multiple DCs, and Offices all connected on the backend via a 'fiber' connection. I am thinking of splitting our different sites out as tenants, and was wondering about the above connectivity documentation.

I have not attempted Tenancy within Netbox yet, was just curious.

Thanks,
Dirk

**Comment by candlerb** [comment]
You can connect a cable between any two ports or interfaces.  It doesn't care what tenancy has been set on the devices that those ports or interfaces reside within.
